amethyst_gltf/src/format/mod.rs

```diff
@@ -285,9 +286,13 @@ fn load_node(
     // Load transformation data, default will be identity
     let (translation, rotation, scale) = node.transform().decomposed();
     let mut local_transform = Transform::default();
-    local_transform.translation = translation.into();
-    // gltf quat format: [x, y, z, w], our quat format: [w, x, y, z]
-    local_transform.rotation = [rotation[3], rotation[0], rotation[1], rotation[2]].into();
+    *local_transform.translation_mut() = translation.into();
+    *local_transform.rotation_mut() = Unit::new_normalize(Quaternion::new(
+        rotation[0],
+        rotation[1],
+        rotation[2],
+        rotation[3],
+    ));
```

Comment:
The comment `gltf quat format: [x, y, z, w], our quat format: [w, x, y, z]` still applies to the `Quaternion::new` constructor of nalgebra. Thus it should be:

```rust
// gltf quat format: [x, y, z, w], argument order expected by our quaternion: (w, x, y, z)
*local_transform.rotation_mut() = Unit::new_normalize(Quaternion::new(
   rotation[3],
   rotation[0],
   rotation[1],
   rotation[2],
));
```

This will fix the `gltf` example.